specification/ai/Face/routes.common.tsp

```diff
@@ -0,0 +1,278 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "./models.common.tsp";
+import "./models.detect.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.Core.Traits;
+using Foundations;
+
+namespace Face;
+
+alias ServiceTraits = NoClientRequestId &
```

Comment:
Why do you say you don't support a client request ID here but then have a property for it in your audit models? We do recommend supporting client request IDs to assist callers with tracing, etc.

Reply:
We support it but not with header key "x-ms-client-request-id", which is the behavior of SupportsClientRequestId trait.